### PR TITLE
Tidy pyproject.toml

### PR DESCRIPTION
**Pull request by alexdewar**
I happened to notice there are a few small mistakes in the `pyproject.toml` file, so I've fixed them up.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
-name = "rseblog"
+name = "RSE Blog"
 version = "0.1.0"
 description = "Blog for Imperial's Central RSE Team"
-authors = ["Your Name <you@example.com>"]
+authors = ["Imperial College London RSE Team <ict-rse-team@imperial.ac.uk>"]
```
Comment (AdrianDAlessandro) [inline pyproject.toml:5]: This is quite funny

**Review by AdrianDAlessandro — APPROVED** [review]
Looks good, need to update the lock file though

**Comment by alexdewar** [comment]
> Looks good, need to update the lock file though

Oops! Indeed I do

**Review by HarmonicReflux — COMMENTED** [review]
Smooth.

**Review by HarmonicReflux — COMMENTED** [review]
Great.

**Pull request merged**